packages/api/src/libs/osnap/utils.ts

```diff
@@ -0,0 +1,28 @@
+import assert from "assert";
+import { Json } from "../../types";
+import type { OsnapPluginData, Transaction } from "./interfaces";
+import { simulateTenderlyTx, TenderlySimulationParams } from "@uma/common";
+
+export async function simulateOsnapTx(space: OsnapPluginData, transactionIndex = 0): Promise<Json> {
+  const { safe } = space;
+  assert(safe, "Requires a safe");
+  const { network, transactions } = safe;
+  assert(transactions.length > transactionIndex, "Cannot simulate not existent transaction");
+
+  // we only expect a single tx
+  const transaction = transactions[transactionIndex];
+  const simParams = mapOsnapTxToTenderlySim(transaction, Number(network));
+  // we are passing the result straight to express, so we are casting this as json. this is fully
+  // compatible with json type since all values are primitives.
+  return ((await simulateTenderlyTx(simParams)) as unknown) as Json;
+}
+export function mapOsnapTxToTenderlySim(tx: Transaction, chainId: number): TenderlySimulationParams {
+  return {
+    chainId,
+    from: "0x0000000000000000000000000000000000000000",
```

Comment:
This is not strictly required. TenderlySimulation helpers from common use zero address by default if from is not provided